### DISCUSSION TITLE
Requirement to start one program before another

### DISCUSSION THREAD

**Discussion by markrich-aww** [kimmknight/remoteapptool#118] [Q&A]
Is there a helpful way to run one program on the remote PC before starting another?

While the remote app works well, some need a pre-requisite app to be started first, which can only be done by a full RDP session to the PC before closing that and then using the remote app.

**Comment by MrBrianGale** [comment]
That's a tricky one. The EASIEST way that I can think of is to have the app that needs to be running first be set up as a windows service so it is always running. If that is not an option, then you'd need to either create 2 remote apps (one for each app that needs to be running) OR modify the code behind the "main" app so it can start up all other required apps first.

RemoteAppTool, nor the Microsoft solution, has no ability to "start" one app prior to starting another.
